docs/features/enterprise/argocd.md

```diff
@@ -0,0 +1,39 @@
+# ArgoCD
+
+Argo CD is a declarative, GitOps continuous delivery tool for Kubernetes. It follows the GitOps pattern of using Git repositories as a source for defining the desired application state. Argo CD automates the deployment of the desired application states in the specified target environments. Application deployments can track updates to branches, tags, or pinned to a specific version of manifests at a Git commit. For more details check out [ArgoCD official documentation](https://argoproj.github.io/argo-cd/)
+
+
+**Setting up ArgoCD**
+
+In order to set up ArgoCD to work with Kubernetic you, firstly, need to install the Operation Lifecycle Manager (OLM). See [Installing OLM](https://docs.kubernetic.com/#/features/settings/addons?id=installing-the-operator-lifecycle-manager-olm).
+
+After you have the OLM installed, go to the "Available Operators" tab, select "argocd-operator" and press "Install"
+
+![](../images/install-argo.png)
+
+You should have a namespace called **my-argocd-operator** created and, in the side bar, the menu caled **ArgoCD** should have appeared. 
```

Comment:
> You should have a namespace called **my-argocd-operator** created...

it should be better clarified that this will be created during installation process itself, and it's not a requirement.